packages/edit-post/src/components/browser-url/test/index.js

```diff
@@ -67,29 +62,25 @@ describe( 'BrowserURL', () => {
 	} );
 
 	it( 'not update URL if history is already set', () => {
-		const wrapper = shallow( <BrowserURL /> );
-		wrapper.setProps( {
-			postId: 1,
-			postStatus: 'draft',
-		} );
+		const { rerender } = render( <BrowserURL /> );
+
+		rerender( <BrowserURL postId={ 1 } postStatus="draft" /> );
+
+		rerender( <BrowserURL postId={ 1 } postStatus="draft" /> );
+
 		replaceStateSpy.mockReset();
```

Comment:
I think we should `mockReset` before the second rerender, no?

Reply:
Makes sense to me 👍 Moved in https://github.com/WordPress/gutenberg/pull/43978/commits/408b750fcc6b3555d1a795b865f0f2fb6383946c